examples/tracing/python/docker-compose.yaml

```
@@ -0,0 +1,85 @@
+services:
+  pyroscope:
+    image: grafana/pyroscope
+    ports:
+      - "4040:4040"
+
+  rideshare-python-us-east:
```

The existing java one and my #3765 uses just the region name as container name. Not too sure what is better, but we should do the same everywhere:

```suggestion
  us-east:
```